Backpack/src/main/java/net/skyscanner/backpack/button/BpkButtonLink.kt

```diff
@@ -93,5 +94,6 @@ open class BpkButtonLink @JvmOverloads constructor(
       (resources.getDimensionPixelSize(R.dimen.bpkBorderSizeLg) / 2)
 
     setPadding(0, paddingVertical, 0, paddingVertical)
+    BpkText.getFont(this.context, BpkText.TextStyle.Label2).applyTo(this)
```

Comment:
link also needs the large variant, though

Reply:
Lets do in in another ticket

Reply:
I'd rather tackle it as part of the same change as it'd probably have breaking implications to do so later - i.e. moving the enum to the base class